rikaicontent.js

```diff
@@ -750,6 +750,14 @@ var rcxContent = {
 		sel.removeAllRanges();
 		sel.addRange(range);
 		tdata.selText = sel.toString();
+
+		if(window.rikaichan.config.ttsEnabled == 'true') {

```

Comment:
is it annoying if it talks all the time? i wonder if it should be a shortcut or button. I'll test it out as well.

Reply:
I guess we mentioned this before but I just meant that even if it was enabled maybe you don't want every single word spoken. I'll test it though.

Reply:
Sounded pretty good. I'm convinced.